src/pages/about/index.js

```diff
@@ -90,7 +90,7 @@ function About() {
                 )}{' '}
                 <ul>
                     <li>
-                        <a href="https://github.com/kokarn">@kokarn</a>{' (created the original Tarkov-Tools but has no ongoing involvement in this fork) '}
+                        <a href="https://github.com/kokarn">@kokarn</a>{' (created the original Tarkov-Tools but has no ongoing involvement in their fork) '}
```

Comment:
I'm not sure what this is meant to convey; who does the "their" refer to here?

Reply:
I didn't want `this` to imply "the current page" meaning that our site no longer has involvement because that is certainly not the case. Didn't want folks to get confused. `their` would imply the `kokarn` fork

Reply:
I pushed a commit to make the wording more clear